src/bindings/python/src/compatibility/openvino/inference_engine/constants.pyx

```diff
@@ -66,13 +66,21 @@ class ResizeAlgorithm(Enum):
 
 
 class ColorFormat(Enum):
+    @classmethod
+    def _missing_(cls, value: object):
+        """Add deprecation warningt to Field2"""
+        if str(value) == "NV12":
+            print("Field name `NV12` for `ColorFormat` enum is deprecated and will be removed in 2023.1 release.")
+            return 5
+        if str(value) == "I420":
+            print("Field name `I420` for `ColorFormat` enum is deprecated and will be removed in 2023.1 release.")
+            return 6
+        return value
```

Comment:
@jiwaszki @akuporos do you have suggestions how to properly disable this API? or is it ok?